src/routes/userRoute.ts

```diff
@@ -0,0 +1,160 @@
+import { Hono } from 'hono';
+import { PrismaClient } from '@prisma/client';
+import { validator } from 'hono/validator';
+import {
+  schemaEmail,
+  schemaId,
+  schemaLogin,
+  schemaPassword,
+  schemaUser,
+} from '../models/userSchema';
+import bcrypt from 'bcrypt';
+
+const prisma = new PrismaClient();
+
+export const userRoute = new Hono();
```

Comment:
Please add a basePath like this, since it's a user route
`export const userRoutes = new Hono().basePath('/users');`